### PR TITLE
Update demo secret yaml

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

**Review by ArbelNathan — APPROVED** [review]

File: deploy/kubernetes/examples/demo-secret.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ stringData:
   management_address: demo-management-address  # Array management addresses
   username: demo-username                      # Array username
 data:
-  password: ZGVtby1wYXNzd29yZA==               # Array password
+  password: ZGVtby1wYXNzd29yZA==               # Array password - demo-password in base64
```
Comment (oriyarde) [inline deploy/kubernetes/examples/demo-secret.yaml:11]: how about `# base64 encoded Array password`? @rivkap what do you think?

Comment (rivkap) [reply deploy/kubernetes/examples/demo-secret.yaml:11]: does it need "encoded"? if not the following is fine. either way.

# base64 array password

@oriyarde  @lisyukanna

Comment (oriyarde) [reply deploy/kubernetes/examples/demo-secret.yaml:11]: `base64 array password` sounds good to me

Comment (lisyukanna) [reply deploy/kubernetes/examples/demo-secret.yaml]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**